core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/RuntimeInitializedPackageBuildItem.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.deployment.builditem.nativeimage;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class RuntimeInitializedPackageBuildItem extends MultiBuildItem {
```

Comment:
Please add some Javadoc to this class to appear nicely in https://quarkus.io/guides/all-builditems